src/Tablebot/Plugins/Basic.hs

```diff
@@ -58,6 +58,10 @@ basicCommands =
     ( "about",
       "This bot was created by finnbar to replace a couple of other bots in Tabletop. It's written in Haskell, and you can find the github here: <https://github.com/WarwickTabletop/tablebot>. There are setup guides and a contributor's guide to help you get started.",
       Simple ("some information about the bot", "Some information about the bot, including how you can get involved")
+    ),
+    ( "inventory",
+      "Here's the board games we possess! -> https://www.warwicktabletop.co.uk/inventory/boardgames/",
+      Simple ("the inventory", "The inventory, with a link to the actual inventory")
```

Comment:
```suggestion
      Simple ("our board games inventory", "Our board games inventory, with a link to the actual inventory")
```
Tiny edit for clarification there - we technically have other inventories, but they're used a lot less.